sound/soc/intel/boards/sof_maxim_common.c

```diff
@@ -10,67 +10,80 @@
 #include <sound/soc-dapm.h>
 #include <uapi/sound/asound.h>
 #include "sof_maxim_common.h"
+#include <sound/pcm_params.h>
 
 #define MAX_98373_PIN_NAME 16
+#define MAXIM_HID_NUMS 3
+#define MAXIM_MAX_AMPS 4
+#define MAXIM_NUM_CODECS 2
+
+static char maxim_codec_name[SND_ACPI_I2C_ID_LEN];
+static struct snd_soc_codec_conf maxim_codec_conf[MAXIM_MAX_AMPS];
+static struct snd_soc_dai_link_component maxim_components[MAXIM_MAX_AMPS];
+
+static const char * const maxim_name_prefixes[] = { "Right", "Left", "Tweeter Right", "Tweeter Left" };
+static unsigned int num_codecs = 2;
```

Comment:
not idea what this means.

Reply:
@plbossart I rewrote it simply I thought. Please help review.